drivers/sensor/lis2dh/lis2dh.c

```diff
@@ -323,18 +267,34 @@ static const struct sensor_driver_api lis2dh_driver_api = {
 int lis2dh_init(struct device *dev)
 {
 	struct lis2dh_data *lis2dh = dev->driver_data;
+	const struct lis2dh_config *cfg = dev->config->config_info;
 	int status;
+	u8_t id;
 	u8_t raw[6];
 
-	status = lis2dh_bus_configure(dev);
+	lis2dh->bus = device_get_binding(cfg->bus_name);
+	if (!lis2dh->bus) {
+		LOG_DBG("master not found: %s", cfg->bus_name);
```

Comment:
Recommend changing this to `LOG_ERR`

Reply:
Done, but ... what's the rule behind it? The errno (EINVAL in this case)?